### PR TITLE
Visitor can read an article

### PR DESCRIPTION
**Pull request by Carrosen**
# [PT Story](https://www.pivotaltracker.com/story/show/166154816)

## Changes proposed in this PR
* Creates a component where the user can read a full article after clicking on it on the landing page
* Makes the Glocal News logo clickable which redirects to the landing page

## What we have learned
* Learning about React router DOM
* How to pass state to another component with redirect
* React dev tools extension for chrome are not our enemy
* Set state is an asynchronous command

Screenshot from 2019-05-27 09-28-26

### CODE REVIEW

File: cypress/integration/visitorCanReadAnArticle.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+describe('Visitor can read an article', () => {
+
+  it('shows full article when clicked', () => {
+    cy.server();
+    cy.route({
```
Comment (oliverochman) [inline cypress/integration/visitorCanReadAnArticle.spec.js:5]: Add the stubbing and `cy.visit` to a before block

File: src/Components/FullArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import { Divider, Segment, Header, Image, Container, Grid } from 'semantic-ui-react'
+import moment from 'moment'
+
+class FullArticle extends Component {
+  constructor(props) {
```
Comment (oliverochman) [inline src/Components/FullArticle.jsx]: you can simplify this, please use:
```
state = {
      id: '',
      title: '',
      ingress: '',
      body: '',
      image: '',
      written_by: '',
      created_at: ''
}
```

File: src/Components/ListOfArticles.jsx [hunk 3]
```diff
@@ -1,37 +1,60 @@
 import React, { Component } from 'react';
 import axios from 'axios';
 import { Grid, Container } from 'semantic-ui-react'
+import { Redirect } from 'react-router-dom'
 
 class ListOfArticles extends Component {
-  state = {
-      articles: []
+  constructor(props) {
```
Comment (oliverochman) [inline src/Components/ListOfArticles.jsx]: why did you change to another syntax here?

File: src/Components/ListOfArticles.jsx [hunk 4]
```diff
@@ -1,37 +1,60 @@
...
-            </div>
-          )
-        })}
-      </div>
-    ) : (
-      <h2>There are no articles at the moment. You can be the first to post your own article and become a neighborhood journalist!</h2>
-    )
+    let articleList
+    if (this.state.redirect === true) {
+      return <Redirect to={{
```
Comment (oliverochman) [inline src/Components/ListOfArticles.jsx]: Why do you introduce the `Redirect` component here instead of using `Link` as in other parts of your code.

IMO, that would be cleaner and you could remove the if statement that you added to this component

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: cypress/integration/visitorCanReadAnArticle.spec.js (hunk 1 above)
Comment (raptorf1) [reply cypress/integration/visitorCanReadAnArticle.spec.js:5]: done

File: src/Components/FullArticle.jsx (hunk 2 above)
Comment (raptorf1) [reply src/Components/FullArticle.jsx]: done

File: src/Components/ListOfArticles.jsx (hunk 3 above)
Comment (raptorf1) [reply src/Components/ListOfArticles.jsx]: testing. changed again. thanks.

File: src/Components/ListOfArticles.jsx (hunk 4 above)
Comment (zanenkn) [reply src/Components/ListOfArticles.jsx]: Changed to Link

**Review by oliverochman — APPROVED** [review]
Good job guys
